src/components/PageSizeSelect/PageSizeSelect.tsx

```diff
@@ -0,0 +1,31 @@
+enum SelectValue {
+  THREE = 3,
+  FIVE = 5,
+  TEN = 10,
+  TWENTY = 20,
+}
+
+type Props = {
+  perPage: number
+  onChangeSize: (event: React.ChangeEvent<HTMLSelectElement>) => void
+};
+
+export const PageSizeSelect: React.FC<Props> = ({
+  perPage,
+  onChangeSize,
+}) => {
+  return (
+    <select
+      data-cy="perPageSelector"
+      id="perPageSelector"
+      className="form-control"
+      defaultValue={perPage}
+      onChange={(onChangeSize)}
```

Comment:
You should pass the onChange function directly to the onChange prop, instead of wrapping it in an anonymous function.